hudi-common/src/main/java/org/apache/hudi/common/table/view/AbstractTableFileSystemView.java

```diff
@@ -170,15 +176,25 @@ public List<HoodieFileGroup> addFilesToView(FileStatus[] statuses) {
 
   /**
    * Build FileGroups from passed in file-status.
+   * TODO: Note here timeline is completedAndCompactionTimeline.
```

Comment:
Yeah, makes sense. We can also avoid doing multiple times filtering on the activeTimeline to get this timeline. So, renamed in all the places so that developers dont even have to perform extra filtering.